### DISCUSSION TITLE
Zasoby /parcel-management-settings/address_book lub/oraz /users-with-processes

### DISCUSSION THREAD

**Discussion by JamesFordV3** [allegro/allegro-api#10063] [Suggestions - resolved]
Cześć,

Tak jak w temacie (punkt 2):  #10055

Prosiłbym o udostępnienie przynajmniej jednego z zasobów do API:
GET `/parcel-management-settings/address_book`
GET `/users-with-processes`

Tak jak w wspomnianym temacie wnioskuję o ten zaspób ponieważ umożliwiłoby mi to dynamiczne wypełnianie danych nadawcy podczas tworzenia przesyłki. Byłoby dobrze aby przynajmniej pierwszy zasób został dodany do API.

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię. Jeśli zdecydujemy się na wprowadzenie takich zasobów, będziemy informować na naszej stronie i forum.

**Comment by MaciejFrackowiak** [comment]
Zoptymalizowaliśmy proces nadawania przesyłek poprzez [wprowadzenie proponowanych danych i ustawień dostawy](https://developer.allegro.pl/news/wysylam-z-allegro-optymalizujemy-proces-nadania-przesylki-od-dzisiaj-dla-kazdego-zamowienia-pobierzesz-proponowane-dane-i-ustawienia-dostawy-potrzebne-do-nadania-przesylki-zlOzlovz3Ia), już na zasobach /shipment-management.
